index.html

```diff
@@ -23,7 +23,7 @@ <h2>List of items</h2>
       <tr>
         <th>Name</th>
         <th style="width:10%">Quantaty</th>
-        <th style="width:10%"></th>
+        <th style="width:30%"></th>
```

Comment:
This is might be the reason why we saw column with buttons wider than we expected. I would keep it 10% or even less, then the browser will try to keep it as narrow as possible.